staticfiles/templates/authentication/register.html

```diff
@@ -2,38 +2,60 @@
          <div class="row text-center">
             <h1 style="font-size: 45px;">Crowd Computing Registration</h1>
             <p>Your Journey Begins Here</p>
-            <hr>       
+            <hr>  
+     <a ><img src="/static/images/activeTab.png"></a>     
       <div class="row">
         <div class="col-md-4 col-md-offset-4">
           <div class="well">
             <form role="form" ng-submit="vm.register()">
+	      <div class="form-group">
+                <label  for="register__country">Country of Residence</label>
+                <!--<input type="text" />-->
+		<select class="form-control" id="country" ng-model="vm.country" placeholder="Country"  ">
+		</select>
+    		<script>$.ajax({
+		    url:'http://localhost:8000/static/templates/authentication/country.json',
```

Comment:
this can be accomplished without using jQuery :) ideally using angular, we should not use jQuery unless it cannot be accomplished otherwise try doing this:
https://docs.angularjs.org/api/ng/directive/select